packages/bash.rb

```diff
@@ -0,0 +1,18 @@
+require 'package'
+
+class Bash < Package
+  description 'The Bourne Again SHell'
+  homepage 'http://www.gnu.org/software/bash/'
+  version '5.1-rc2'
+  source_url 'http://ftp.gnu.org/gnu/bash/bash-5.1-rc2.tar.gz'
```

Comment:
Use https?

Reply:
Actually, it's probably better to use a mirror:
```
source_url 'https://ftpmirror.gnu.org/bash/bash-5.1-rc2.tar.gz'
```

Reply:
Done